packages/diagrams/frontend/sirius-components-diagrams-reactflow/src/graphql/subscription/nodeFragment.ts

```diff
@@ -62,6 +62,8 @@ fragment nodeFragment on Node {
     }
   }
   userResizable
```

Comment:
Is this one made obsolete by the version fetched via `GQLNodeDescription`?

Reply:
Yes, you're right. I removed it from this fragment. But it's still used in the sprotty diagram, so I left it in the graphql API